django/views/templates/technical_500.html

```diff
@@ -195,11 +199,11 @@ <h3>{{ template_info.message|force_escape }}</h3>
       {% if template_info.bottom != template_info.total %} cut-bottom{% endif %}">
    {% for source_line in template_info.source_lines %}
    {% if source_line.0 == template_info.line %}
-   <tr class="error"><th>{{ source_line.0 }}</th>
+   <tr class="error"><th scope="row">{{ source_line.0 }}</th>
```

Comment:
I don't think that the line number in templates can be treated as a header. IMO, we should change it to `<td>`.

Reply:
See https://github.com/django/django/pull/17787#discussion_r1469336676

Reply:
Leaving it as-is per the discussion on the other, similar table above.